dbms/programs/local/LocalServer.cpp

```diff
@@ -77,7 +77,10 @@ void LocalServer::applyCmdSettings(Context & context)
 /// If path is specified and not empty, will try to setup server environment and load existing metadata
 void LocalServer::tryInitPath()
 {
-    std::string path = config().getString("path", "");
+    std::string path;
+    if (!config().has("path") || (path = config().getString("path")).empty())
```

Comment:
`getString` has default parameter.

Reply:
Why we check for empty string?
If the user write `<path></path>` it's their fault.

Reply:
Why should `clickhouse-local` save anything to /var/lib/clickhouse?